README.md

```diff
@@ -0,0 +1,120 @@
+# nextcloud collection for ansible
+
+This repository hosts the `aalaesar.nextcloud`  Ansible Collection (formerly the role `aalaesar.install_nextcloud`).
+
+The collection includes a variety of Ansible content to help automate the management of nextcloud, as well as provisioning and maintenance of instances of nextcloud.
+
+<!--start requires_ansible-->
+## Ansible version compatibility
+
+This collection has been tested against following Ansible versions: **>=2.10.0**.
+
+Plugins and modules within a collection may be tested with only specific Ansible versions.
+<!--end requires_ansible-->
+
+## Python Support
+
+* Collection tested on 3.10+
+
+Note: Python2 is deprecated from [1st January 2020](https://www.python.org/doc/sunset-python-2/). Please switch to Python3.
```

Comment:
@aalaesar i would remove note over Py2 in 2023 actually.

Reply:
Good finding ! Just removed it